docs/tools/offline/cli-device-management.md

```diff
@@ -87,3 +93,5 @@ Once you execute `mbed dm update prepare`, Mbed CLI automatically uploads the pa
 ### Advanced use
 
 Mbed CLI allows for significantly more flexibility than the model above shows in exactly the same way as [the manifest tool](https://cloud.mbed.com/docs/current/updating-firmware/manifest-tool.html). You can override each of the defaults that `mbed dm init` sets by using the command-line or an input file. Mbed CLI supports a variety of commands. You can print a full list of commands by using `manifest-tool --help`.
+
+To test this feature, you can run the [Device Management Client Example](https://github.com/ARMmbed/mbed-cloud-client-example) on one of our reference platforms.
```

Comment:
version 2.0.0 or later. (2.0.0 is not released yet).

I would also link to the Cloud-side tutorial.

Reply:
Why is this mentioned in Advanced usage? This is the only possible way to use the dm feature.

Already in requirements you should link to Cloud-side documentation + quick-starts when those are ready.